akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2Blueprint.scala

```diff
@@ -51,6 +59,7 @@ object Http2Blueprint {
    */
   def httpLayer(): BidiFlow[HttpResponse, Http2SubStream, Http2SubStream, HttpRequest, NotUsed] = {
     val incomingRequests = Flow[Http2SubStream].via(new HttpRequestHeaderHpackDecompression)
+      .log(getClass.getSimpleName)
```

Comment:
Hm, right on the entire blueprint would be better perhaps hm...
Reason to have this is that otherwise errors (onError) literally get swallowed making debugging harder.